frontend/views/containers/proposals/RemoveMember.vue

```diff
@@ -64,11 +66,12 @@ export default {
   flex-direction: column;
   align-items: center;
   margin-bottom: $spacer-sm;
-}
 
-.c-avatar {
-  width: 5rem;
-  height: 5rem;
-  margin-bottom: $spacer;
+  // BUG on VUE? - without nesting it doesn't work
+  .c-avatar {
+    width: 5rem;
+    height: 5rem;
```

Comment:
@pieer and @taoeffect are you aware of this type of issue? For some reason, Vue (or some grunt plugin) extract the CSS from Avatar.vue component _after_ RemoveMember.vue and it should be the other way around. (because Avatar is a dependency of RemoveMember).

For that reason, I had to nested `.c-avatar` overrides the original avatar size.

Note: This shouldn't be a blocker for this PR, but something to be addressed in another PR.

Reply:
@sandrina-p I don't understand the issue well enough to understand the problem.

> Note: This shouldn't be a blocker for this PR, but something to be addressed in another PR.

Can you create an issue then?